packages/components/src/Modal/Modal.test.tsx

```diff
@@ -5,6 +5,11 @@ import styles from "./Modal.css";
 
 afterEach(cleanup);
 
+test('modal contains aria role of "dialog"', async () => {
+  const { findByRole } = render(<Modal open>Content</Modal>);
+  expect(await findByRole("dialog")).toBeDefined();
```

Comment:
```suggestion
  expect(await findByRole("dialog")).toBeInTheDocument();
```

exists here in atlantis 😉